jwst/datamodels/model_base.py

```diff
@@ -88,22 +97,13 @@ def __init__(self, init=None, schema=None, extensions=None,
         self._strict_validation = self.get_envar("STRICT_VALIDATION",
                                                  strict_validation)
 
-        # Construct the path to the schema files
-        filename = os.path.abspath(inspect.getfile(self.__class__))
-        base_url = os.path.join(
-            os.path.dirname(filename), 'schemas', '')
-
         # Load the schema files
         if schema is None:
-            schema_path = os.path.join(base_url, self.schema_url)
+            schema_path = os.path.join(URL_PREFIX, self.schema_url)
```

Comment:
Are these two equivalent? One is the URL_PEFIX the other one is a path on the local system.

Reply:
URL_PREFIX is the `http://jwst.stsci.edu/schemas/` string.  `self.schema_url` is `core.schema.yaml` or equivalent.